ted_sws/event_manager/adapters/event_handler_config.py

```diff
@@ -0,0 +1,132 @@
+import abc
+from datetime import datetime, timezone
+from pathlib import Path
+from typing import List, Type, Union
+
+from pymongo import MongoClient
+
+from ted_sws import config
+from ted_sws.event_manager.adapters.event_handler import EventHandler, EventWriterToConsoleHandler, \
+    EventWriterToFileHandler, EventWriterToNullHandler, EventWriterToMongoDBHandler
+from ted_sws.event_manager.adapters.log import ConfigHandlerType
+
+NOW = datetime.now(timezone.utc)
+DEFAULT_LOGGER_NAME = "ROOT"
+DEFAULT_NULL_LOGGER_NAME = "NULL"
+DEFAULT_LOGGER_LOG_FILENAME = "./event_logs/{today}.log"
+# DEFAULT_LOGGER_LOG_FILENAME = "./event_logs/{today}/{hour}/{minute}.log" # EXAMPLE
```

Comment:
remove comments

Reply:
I've left it to show an example of log file naming